image_provisioner/playbooks/roles/repo-install/files/rhel72.repo

```diff
@@ -0,0 +1,29 @@
+[rhel-72]
+name=Prerelease version of RHEL7.2
+baseurl=https://mirror.openshift.com/enterprise/rhel/rhel7next/os
```

Comment:
I might be able to pull that repo file from the aos-ansible repo as well... Once I get that in before git